### DISCUSSION TITLE
After updating to 0.4.6, the interface for adding models disappeared

### DISCUSSION THREAD

**Discussion by rigin** [open-webui/open-webui#7481] [Issues]
Hello, after updating to the latest version, I lost the interface for adding models to the menu -"Settings" - "Administrator" - "Models". 

Now there is only a list of models.

How do I add models from GGUF archives now?

**Comment by JohnathanCrow** [comment]
Settings > Administrator > Connections > Ollama API > Manage (spanner icon) > Experimental > Upload a GGUF model. 

It's a little bit hidden.

**Comment by rigin** [comment]
How do you open it?

**Reply by JohnathanCrow** [reply]
I'm not sure what you mean? If you go to where I said you should be able to to click on 'Click here to select' and find your GGUF.

**Comment by rigin** [comment]
Oh, I get it, I didn't notice the top line in your message.))
